### PR TITLE
Milestone 1: Set up hello microverse project

### PR DESCRIPTION
**Pull request by VivianDoreen**
## Set up a "Hello Microverse" project
In this project, I
  - Added a .gitignore file
  - Setup Github actions, a linter for HTML and CSS
  - Added a descriptive README file
  - Added an index.html file
  - Add styles.css file
  - Added test.md file with the text "This file should be ignored by git".
  - Modified .gitignore file to include test.md file

### CODE REVIEW

File: .github/workflows/linters.ym [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Stylelint
+        run: |
+          npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+          [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}
```
Comment (Emmyn5600) [inline .github/workflows/linters.ym:48]: - [ ]There is typo in the linters configuration, please change the `linters.ym` with the `linters.yml`

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @VivianDoreen 

Great work on implementing the project requirements, however there are some issues to be solved before you get approved but you are almost there:muscle:. 
   
Changes required :writing_hand:  

 * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: .github/workflows/linters.ym (hunk 1 above)
Comment (VivianDoreen) [reply .github/workflows/linters.ym]: Hello Emmy NSABIMANA, thanks for the feedback,  I have rectified the typo. Kindly review for the changes

**Review by AshakaE — APPROVED** [review]
### Hi  @VivianDoreen  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**